axelor-account/src/main/resources/views/Journal.xml

```diff
@@ -154,5 +155,7 @@
   <action-method name="action-journal-method-set-totals">
     <call class="com.axelor.apps.account.web.JournalController" method="computeBalance"/>
   </action-method>
-
+  <action-method name="action-journal-method-change-config-values">
+    <call class="com.axelor.apps.account.web.JournalController" method="changeConfig"/>
+  </action-method>
```

Comment:
Now that I think about it, it would be so much easier to use an action-record like this:
``` xml
  <action-record name="action-journal-record-change-config-values"
    model="com.axelor.apps.account.db.Journal">
    <field name="authorizeSimulatedMove" expr="eval: company?.accountConfig?.isActivateSimulatedMove"/>
    <field name="allowAccountingDaybook" expr="eval: company?.accountConfig?.accountingDaybook"/>
  </action-record>
```
Also with your action-method, if the user delete the company field, the method will be called and they will get an NPE line 93.